testScripts/compareVersions/CMakeLists.txt

```
@@ -0,0 +1,46 @@
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/${CMAKE_SYSTEM_NAME}-${CMAKE_SYSTEM_PROCESSOR}")
+
+find_program(AWK awk mawk gawk)
+find_program(SORT sort)
+find_program(GNUPLOT gnuplot)
+find_program(CONVERT convert)
+find_program(VLFEAT sift PATH_SUFFIXES glnxa64 bin/glnxa64)
```

```suggestion
find_program(VLFEAT sift HINTS ${VLFEAT_BIN} PATH_SUFFIXES glnxa64 bin/glnxa64 REQUIRED)
```
It's better to have REQUIRED so it stops if it does not find it and `${VLFEAT_BIN}` can be use as hint to find vlfeat